### ISSUE TITLE
Several rs combined be

### ISSUE THREAD

**Issue by dashainparis** [konradjk/exac_browser#296]

**Comment by dashainparis** [comment]
I noticed that in vcf file, ExAC stores different variations with different rsIDs by one rsID.
Example:

quote from DBsnp vcf file (37 genome version):

1	32128283	rs745892055	C	CA,CCCCCG	.	.	RS=745892055;RSPOS=32128283;dbSNPBuildID=144;SSR=0;SAO=0;VP=0x050000080005000002000200;GENEINFO=COL16A1:1307;WGT=1;VC=DIV;INT;ASP
1	32128283	rs1568396	C	A,G	.	.	RS=1568396;RSPOS=32128283;dbSNPBuildID=88;SSR=0;SAO=0;VP=0x05010008000517013e000100;GENEINFO=COL16A1:1307;WGT=1;VC=SNV;SLO;INT;ASP;VLD;G5A;G5;GNO;KGPhase1;KGPhase3;CAF=0.4902,.,0.5098;COMMON=1

quote from ExAC:

1    32128283    rs1568396    C    G,CCCCCG,A,CA    2768259.72    PASS    AC=15561,603,185,4;AC_AFR=358,0,18,0;AC_AMR=51,0,1,0;AC_Adj=3531,0,25,1;AC_EAS=181,0,0,0;AC_FIN=4,0,0,0;AC_Het=1603,0,6,0,0,17,1,0,0,0;AC_Hom=955,0,1,0;AC_NFE=1391,0,2,1;AC_OTH=43,0,1,0;AC_SAS=1503,0,3,0;AF=0.458,0.018,5.447e-03,1.178e-04;AN=33964;AN_AFR=532;AN_AMR=114;AN_Adj=7246;AN_EAS=244;AN_FIN=8;AN_NFE=2472;AN_OTH=84;AN_SAS=3792 ...

So basically one position is associated with several different types of variations simple SNP and Insertions. Each type of variation has it's own rsID (rs745892055,rs1568396) and in ExAC you mix together both type in one variation and use only one rs1568396 without mention of rs745892055.
